src/main/java/gg/projecteden/nexus/models/freeze/Freeze.java

```diff
@@ -72,6 +73,7 @@ public void unfreeze() {
 
 	private void execute() {
 		frozen = true;
+		location = getPlayer().getLocation();
```

Comment:
`getPlayer()` may be null. you should use `getOnlinePlayer()`, or even better in this case, `getLocation()`